.svglintrc.js

```diff
@@ -569,6 +569,31 @@ module.exports = {
                 reporter.error(`${errorMsg}: ${reason}`);
               }
             }
+          },
+          function(reporter, $, ast) {
+            reporter.name = 'svg-format';
+
+            // Don't allow explicit '</path>' closing tag
+            const explicitClosingPathIndex = ast.source.indexOf('</path>');
+            if (explicitClosingPathIndex > -1) {
+              const reason = `found explicit closing "path" tag at index ${explicitClosingPathIndex}.`
+                          + ' The syntax \'></path>\' should be replaced by \'/>\'.';
+              reporter.error(`Invalid SVG content format: ${reason}`);
+            }
+
+            // Don't allow spaces before implicit closing tags
+            const spacedImplicitClosingTagIndexes = [];
+            let index = ast.source.indexOf(' />', -1);
```

Comment:
what if an SVG has `../>` (i.e. with two spaces)? Looks like the reported index would be off by one 🤔

Reply:
In that case, the next time the linter will also trigger this rule.

Reply:
I see. I personally think a linter should not have to run multiple times to tell you all the things that are currently wrong with the code but if I'm the only one that thinks so we can leave it like this.

Reply:
>  I personally think a linter should not have to run multiple times to tell you all the things that are currently wrong with the code

This also happen with other rules as well. As far as I know we are not linting the possible future changes that we are requiring to the user to make pass the current SVG.

Reply:
I've removed this part of the rule because is now triggered by the `extraneous` rule.